server/src/model.ts

```diff
@@ -1611,7 +1776,7 @@ export async function searchSharedContent({
 }
 
 // TODO: add tests of this api if we're sure we want to keep it
-export async function browseSharedContent({
+export async function browsePublishedLibraryContent({
```

Comment:
Combine with `browseSharedContent` using a parameter to change the where clause.